frontend/src/pages/ApplicationForm/ApplicationOptions.jsx

```diff
@@ -0,0 +1,59 @@
+export const appFormStep = [
+  {
+    num: 1,
+    class: 'personalInfo',
+    icon: 'user',
+    title: 'Personal',
+    description: 'Please enter in your personal information'
+  },
+  {
+    num: 2,
+    class: 'academicInfo',
+    icon: 'industry',
+    title: 'Academic',
+    description: 'Please enter in your academic information'
+  },
+  {
+    num: 3,
+    class: 'mentorshipInterest',
+    icon: 'tag',
+    title: 'Interest',
+    description: 'Enter in mentorship interest'
+  }
+];
+
+export const genderOptions = [
+  { key: '', text: '', value: '' },
+  { key: 'm', text: 'Male', value: 'M' },
+  { key: 'f', text: 'Female', value: 'F' }
+];
+export const gradeLevelOptions = [
+  { key: 'hs', text: 'High School / Secondary', value: 'high_school' },
+  { key: 'under', text: 'Undergrad', value: 'undergraduate' },
+  { key: 'ex', text: 'Exchange Student', value: 'exchange' },
+  { key: 't', text: 'Transfer Student', value: 'transfer' },
+  { key: 'g', text: 'Graduate Student', value: 'graduate' }
+];
+export const referralOptions = [
+  { key: 'f', text: 'Friend', value: 'friend' },
+  { key: 't', text: 'Teacher', value: 'teacher' },
+  { key: 'p', text: 'Parent', value: 'parent' },
+  { key: 's', text: 'School or Prep School', value: 'school' },
+  { key: 'sm', text: 'Social Media', value: 'socialMedia' },
+  { key: 'i', text: 'Internet', value: 'internet' },
+  { key: 'o', text: 'Other', value: 'other' }
+];
+
+export const topicsOptions = [
+  { key: 'e', text: 'Practicing / Improving English', value: 'english' },
+  { key: 'fr', text: 'Making American Friends', value: 'friends' },
+  {
+    key: 'c',
+    text: 'Connecting with other International Students',
+    value: 'connecting'
+  },
+  { key: 'j', text: 'Getting a Job / Internship in America', value: 'job' },
+  { key: 'a', text: 'Adjusting to American Culture', value: 'culture' },
+  { key: 't', text: 'Travelling', value: 'travelling' },
+  { key: 'o', text: 'Other', value: 'other' }

```

Comment:
Are these keys correlated to stuff in the database?  Will the database items be stored as these 1-2 char strings, or as some sort of ID?

Reply:
I think the keys are just for React, maybe @jessewoo can comment further on that but one thing to note is that topicsOptions is not really functional right now, because we didn't want to invest time on making a checklist and a list type field in the DB since it's not a super essential piece of information to have. So currently there is a string field that we can use to store the list CSV style. When we do make this work, what will be sent back is the value field.

Reply:
@kailanjian is right. keys need to be acted for React, a unique key is used to quickly search for what has been changed on the virtual DOM. https://medium.com/@adhithiravi/why-do-i-need-keys-in-react-lists-dbb522188bbb